x-pack/plugins/observability_solution/observability_ai_assistant_app/public/components/nav_control/index.tsx

```diff
@@ -41,6 +42,19 @@ export function NavControl({}: {}) {
     [service, hasBeenOpened]
   );
 
+  useEffect(() => {
```

Comment:
is L39 not a better place to handle it?

Reply:
you can add a catch handler before the value is returned though, e.g. ` service.start({ signal }).catch(notifyError)`

Reply:
Fixed in https://github.com/elastic/kibana/pull/179335/commits/60dea266b2c46093f999109c2d863654d7d205f6

Reply:
Is it just because it returns a Promise (since it's `async`) and then that error bubbles up as part of the Promise execution which would be out side of the execution context of the `catch` already?

Reply:
https://github.com/elastic/kibana/pull/179335/commits/25f02a71b3a7bec88a63df33dc2a313c1694de08